contrib/slurm-test/sort.py

```diff
@@ -0,0 +1 @@
+/home/lukas/Desktop/toil/src/toil/test/sort/sort.py
```

Comment:
I don't think this symlink will work. It should be relative, not an absolute path